src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -129,12 +135,19 @@ public int parseArguments(Parameters params) throws CmdLineException {
   )
   String skipEmitRegExp = null;
 
+  @Option(
+    name = "--googProvides",
+    usage = "file containing a list of module ids to treat as goog.provide style modules"
```

Comment:
'module ids' can be confusing, as they don't come from modules. I would say 'namespaces names that we know come from goog.provides (not goog.modules)'.